inference-engine/tests/unit/gna/ngraph/transformations/gna_split_convolution_with_large_buffer_size.cpp

```diff
@@ -288,7 +288,8 @@ template <typename TransformationT>
 ngraph::pass::Manager createPassManager() {
     ngraph::pass::Manager manager;
     manager.register_pass<ngraph::pass::InitNodeInfo>();
-    manager.register_pass<TransformationT>();
+    auto transformation = manager.register_pass<TransformationT>();
+    transformation->SplitAll();
```

Comment:
I'd modify the test to work with 1d convolutions only since the transformation isn't currently adapted for 2d convolutions. So I don't think that it will be correctly to force it to split all convolutions.

Reply:
I reverted this